### ISSUE TITLE
PB2NC bug using observations of virtual temperature instead of sensible temperature.

### ISSUE THREAD

**Issue by dwfncar** [dtcenter/MET#309]
The PB2NC tool was selecting out observations of virtual temperature and writing them to the output NetCDF file.  For verification, we want sensible temperature instead.  To determine the type of temperature value you have, check the corresponding program code and reason code.  Refer to PREPBUFR Table 14 describes the VIRTMP processing step:
<br/>
   <a href="http://www.emc.ncep.noaa.gov/mmb/data_processing/prepbufr.doc/table_14.htm">http://www.emc.ncep.noaa.gov/mmb/data_processing/prepbufr.doc/table_14.htm</a>
<br/>

<br/>
For VIRTMP program code 8 with reason code 3, do not use this this observation since it&#39;s already virtual temperature.
<br/>
For VIRTMP program code 8 with any other reason code, step up the event index to find sensible temperature, where the program code != 8.
<br/>

<br/>
So basically, we&#39;re stepping up/down the event stack looking for a version of temperature that&#39;s sensible.  This can sometimes result in a different quality marker since the quality marker values change for different places on the event stack.
<br/>

<br/>
The above logic was confirmed by checking with Perry Shafran and Dennis Keyser at NCEP.
<br/>

<br/>

<i>[MET-309] created by johnhg</i>
